.mocharc.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   require: [
     'source-map-support/register',
     'ts-node/register',
+    'test/tools/runner/throw_rejections.cjs',
```

Comment:
We're using this instead of the strict flag because the strict flag causes a SIGABRT locally.